package.json

```
@@ -2,21 +2,20 @@
   "name": "swr",
   "version": "0.4.0",
   "description": "React Hooks library for remote data fetching",
-  "main": "./dist/index.js",
-  "module": "./esm/index.js",
-  "react-native": "./esm/index.js",
-  "types": "./dist/index.d.ts",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.module.js",
```

how about also adding a modern bundle in output? that will be the minimal size of bundle output https://github.com/developit/microbundle#modern

```suggestion
  "module": "dist/index.module.js",
  "mordern": "dist/index.mordern.js",
```

---

It already has `"exports": "dist/index.modern.js",` though, I guess that's why the codesandbox example works? 🤔